src/main/java/com/transferwise/banks/demo/customer/persistence/CustomerEntity.java

```diff
@@ -1,15 +1,13 @@
 package com.transferwise.banks.demo.customer.persistence;
 
 import com.transferwise.banks.demo.client.params.Email;
+import com.transferwise.banks.demo.customer.persistence.address.AddressEntity;
 
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.Id;
+import javax.persistence.*;
 import java.time.LocalDate;
 
 @Entity(name = "customers")
-class CustomerEntity {
+public class CustomerEntity {
```

Comment:
why did we need to change the visibility of this class?

Reply:
I forgot to change it back I think. It was because of the way I was doing the one to one relationship.

Reply:
OK that's strange because in my code it is not public